code-of-conduct.html

```diff
@@ -75,8 +75,8 @@
 				<div class="need-help rounded-corners">
 					<h3>Need Help?</h3>
 					<p>Our contact information will be provided to all conference participants and speakers upon registration.</p>
-					<p>Conference organizers can be contacted by email at:</p>
-					<a class="dont-break-out" href="mailto:info@writespeakcode.com">info@writespeakcode.com</a>
+					<p>Conference organizers can be contacted by email at: <a href="mailto:info@writespeakcode.com">info@writespeakcode.com</a></p>
+
 					<p>If the matter is especially urgent, please call/contact any of these individuals*:</p>
```

Comment:
Basically, I realized that the link style w/ letter spacing of 2px and uppercase was not conducive to mobile screens. Even w/ the don't break out class, I thought it was awkward that a link would wrap that way. I think the don't break out solution works really well for super long links that you want to display the entire URL, for example.